### ISSUE TITLE
New Tab-Button missing

### ISSUE THREAD

**Issue by KN4CK3R** [HeidiSQL/HeidiSQL#1167]
**Describe the bug**
The New Tab-Button disappears sometimes.

**To Reproduce**
1. Open some tabs (see screenshot 1)
2. Select a table. Now two new tabs (Table and Data) appear and the tab bar gets a second row
3. The New Tab-button isn't visible anymore (see screenshot 2)

**Expected behavior**
New Tab-Button should be visible all the time. Ctrl+T still works to create new tabs.

**Screenshots**
New Tab-Button on the right
![grafik](https://user-images.githubusercontent.com/1666336/94117798-f6cfbb00-fe4c-11ea-97ef-a7954cab3a38.png)

Missing New Tab-Button
![grafik](https://user-images.githubusercontent.com/1666336/94117896-18c93d80-fe4d-11ea-82de-189ee7dcbfc6.png)

**Environment:**
 - OS: Windows 10
 - HeidiSQL version 11.0.0.6098

**Comment by WesleyM77** [comment]
I noticed this a while ago as well, but figured it was intended to force you to keep your workspace clean. Then again, who's to tell you how to work? The new tab button disappears as soon as the second line of tabs is made.

**Comment by ansgarbecker** [comment]
No, this is not intentionally, just an ordinary bug. Caused by the fact that multiple tab rows take the full width without leaving space for a button. This is hardly fixable with the current page control.
Note there's always the menu item _Query > New query tab_, and its shortcut <kbd>Ctrl+T</kbd>.
